rocAL/CMakeLists.txt

```diff
@@ -89,6 +89,7 @@ if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
         message("-- ${Yellow}NOTE: rocAL library requires HIP for BACKEND=HIP, Not Found ${ColourReset}")
         message("-- ${Yellow}NOTE: rocAL Backend set to CPU ${ColourReset}")
     endif()
+    find_package(rocDecode QUIET)
```

Comment:
Check if we should move this after line 46.?

Reply:
Done